cookbooks/Gemini/Gemini.ipynb

```diff
@@ -54,7 +54,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 1,
+      "execution_count": null,
```

Comment:
Let's just remove this line. I know it won't error but just good to remove no-op params from cookbooks

Reply:
ipynb grabbed from https://github.com/lastmile-ai/aiconfig/blob/e2f2d191c1367166efa9717305ac26f7a524d74a/cookbooks/Gemini/Gemini.ipynb